viewer/src/models/cad/BasicSectorActivator.ts

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import { setUnion, setDifference } from '../../utils/setUtils';
+import { DiscardSectorDelegate, ConsumeSectorDelegate, GetSectorDelegate } from './delegates';
+
+export interface SectorActivator {
```

Comment:
Should we split interfaces and implementations earlier rather than later? Probably not something we need to look into before we go over architecture, but I think it is important to keep in mind going further.

Reply:
"Split", as in putting them in different files?